corehq/apps/es/case_search.py

```diff
@@ -312,3 +333,69 @@ def flatten_result(hit, include_score=False, is_related_case=False):
     for key in SYSTEM_PROPERTIES:
         result.pop(key, None)
     return result
+
+
+def wrap_case_search_hit(hit, include_score=False, is_related_case=False):
+    """Convert case search index hit to CommCareCaseSQL
+
+    Nearly the opposite of
+    `corehq.pillows.case_search.transform_case_for_elasticsearch`.
+
+    The "case_properties" list of key/value pairs is converted to a dict
+    and assigned to `case_json`. 'Secial' case properties are excluded
+    from `case_json`, even if they were present in the original case's
+    dynamic properties.
```

Comment:
I assume this is because we can't tell the difference anymore between properties inserted via `transform_case_for_elasticsearch` and those set directly on the original?

Reply:
Correct